### PR TITLE
Fix streaming demo env image for k8s

### PR DESCRIPTION
**Pull request by jomrdjan**

### CODE REVIEW

File: jupyterlab-streaming/Dockerfile [hunk 1]
```diff
@@ -1,65 +1,31 @@
-FROM maven:3.8.3-openjdk-11 AS MAVEN_BUILD
-ARG FLINK_DEPS_TMP=/var/tmp/flink-deps/
+FROM gcr.io/getindata-images-public/jupyterlab-base:ubuntu-20.04.python-3.8
```
Comment (p-pekala) [inline jupyterlab-streaming/Dockerfile:1]: why is it based on our public repo. Do we build this jupyterlab-base ?

**Review by p-pekala — APPROVED** [review]

File: jupyterlab-streaming/Dockerfile [hunk 2]
```diff
@@ -1,65 +1,32 @@
-FROM maven:3.8.3-openjdk-11 AS MAVEN_BUILD
-ARG FLINK_DEPS_TMP=/var/tmp/flink-deps/
-
-COPY pom.xml .
-RUN mkdir $FLINK_DEPS_TMP
-RUN mvn dependency:copy-dependencies -DoutputDirectory=$FLINK_DEPS_TMP
-
-########################################################################
 FROM jupyter/base-notebook:python-3.8
```
Comment (jomrdjan) [inline jupyterlab-streaming/Dockerfile:1]: Source of base image was changed to public one

File: jupyterlab-streaming/Dockerfile [hunk 3]
```diff
@@ -1,65 +1,32 @@
...
+COPY resources/jupyter_server_config_append /var/tmp/jupyter/
+RUN cat /var/tmp/jupyter/jupyter_server_config_append >> /etc/jupyter/jupyter_server_config.py && \
+    rm /var/tmp/jupyter/jupyter_server_config_append
+RUN sudo apt update -y && \
+    sudo apt-get install -y git maven openjdk-11-jdk vim curl wget && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /opt/kafka && \
+    chown jovyan:root /opt/kafka && \
+    chmod 770 /opt/kafka
```
Comment (jomrdjan) [inline jupyterlab-streaming/Dockerfile:23]: Permissions changed from 777 to all users, to 770 for jovyan

**Review by jomrdjan — COMMENTED** [review]
Changes done accordingly to our discussion

**Review by p-pekala — APPROVED** [review]

**Pull request merged**